Lib/asyncio/tasks.py

```diff
@@ -255,6 +255,8 @@ def uncancel(self):
         """
         if self._num_cancels_requested > 0:
             self._num_cancels_requested -= 1
+            if self._num_cancels_requested == 0:
+                self._must_cancel = False
```

Comment:
Shouldn't this also clear the cancel message?

Reply:
I doubt it -- `self._cancel_message` is never cleared, and it's never used unless `self._must_cancel` is set.

Reply:
Okay – I was just confused by that `Py_CLEAR()` in the C version, but this explains it.

Reply:
Oh, you're right, that CLEAR is unnecessary there too. :-)